### ISSUE TITLE
Grid Insert Tags should not render in backend

### ISSUE THREAD

**Issue by taleteller** [contao-bootstrap/grid-editor#7]
When using {{grid}} insert tags, they will not only get replaced with div's in the frontend but the backend as well.  Especially when those insert tags are seperated in different elements, the page renders with malformed div's. This results in a broken page structure. 

Example Image attached: 
![Broken Page](https://cloud.githubusercontent.com/assets/1484333/6258014/411a9858-b7c5-11e4-8890-2150ef3baa40.png)

**Comment by dmolineus** [comment]
Since Contao 3.3, insert tags weren't replaced in the backend. Now they are (see contao/core#7428). I was not aware of this (see #7647).

As a workaround you can add following snippet to `system/config/initconfig.php`:

```php

<?php

$GLOBALS['container']['event-dispatcher']->addListener(
        'bootstrap.replace-insert-tags', 
        function($event) {
                if ($event->getTag() == 'grid' && TL_MODE !== 'FE') {
                        $event->setHtml('<span class="tl_gray">GRID INSERTTAG</span>');
                        $event->stopPropagation();
                }
        },
        100
);

```

**Comment by taleteller** [comment]
Yes that overrides the issue, for the grid editor at least. 

However, rendering insert tags in the backend feels like a big invitation for XSS and privilege escalation, since there are worse things to inject than broken divs. But thats for the core team, i guess.

**Comment by dmolineus** [comment]
Fixed in 5216d67e57a0e6291c35e1df996fd3e2696db890.
